<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorAlertDlg</class>
 <widget class="QDialog" name="ColorAlertDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры цветовой индикации</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Метеопараметр:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTreeWidget" name="tree">
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>30</number>
     </attribute>
     <column>
      <property name="text">
       <string>З</string>
      </property>
      <property name="toolTip">
       <string>Зелёный</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>190</red>
        <green>230</green>
        <blue>200</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ж</string>
      </property>
      <property name="toolTip">
       <string>Жёлтый</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>К</string>
      </property>
      <property name="toolTip">
       <string>Красный</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Условие</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="statusLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt; status &gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="okBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Принять</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="cancelBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="paramCombo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>ColorAlertDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
